src/components/Navbar/index.tsx

```diff
@@ -1,6 +1,14 @@
-import React from 'react';
-import './styles.scss';
+import React from "react";
+import "./styles.scss";
+import { Link } from "react-router-dom";
```

Comment:
```suggestion
import React from 'react';
import './styles.scss';
import { Link } from 'react-router-dom';
```

For best practice, we normally use `'` instead of `''` in javascript.
Ref: https://nimblehq.co/compass/development/code-conventions/javascript/

Reply:
To be more specific, use single quotes in `javascript` codes.